.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+jobs:
+  build_and_test:
+    machine: true
+
+    steps:
+    - checkout
+
+    - run:
+        name: Install dep
```

Comment:
Not sure this is needed, actually. Same goes for running `dep ensure`. This should only be needed when developing and changing go imports but checked in versions should be checked with proper, up-to-date vendor / dep files…

Reply:
Understood. I will remove this step then

Reply:
Fixed